tests/parametric/test_otel_span_methods.py

```diff
@@ -535,3 +743,25 @@ def run_otel_span_reserved_attributes_overrides_analytics_event(
     span = get_span(test_agent)
     assert span["metrics"].get("_dd1.sr.eausr") == expected_metric_value
     assert "analytics.event" not in span["meta"]
+
+
+def retrieve_span_links(span):
```

Comment:
It would be nice if we could be explicit about the expected format. This could help us catch regressions:

```suggestion
def retrieve_span_links(span, api="v0.5"):
```

Reply:
I am not very familiar to this. What API refers to?
And what would be the purpose?

Reply:
API refers to the the value set via the `DD_TRACE_API_VERSION` environment variable. It is the the trace encoding format. In python customers can use either `v0.4` or `v0.5`. I think most traces just implement `v0.4` and hide this implementation detail. 

When a span containing links is encoded:
 - With `v0.4` span["span_links"] will be set.
 - With `v0.5` span["meta"]["_dd.span_links"] will be set.
 
This helper already searches for both cases. I think should make this mapping more explicit. This suggestion is honestly just a nit.